airesources/Rust/src/hlt/game.rs

```diff
@@ -41,27 +40,29 @@ impl Game {
 
         println!("{}", name);
 
-        let game = Game {
+        // ignore first line
+        Game::read_line();
```

Comment:
Just to know, why did we need to skip the first map update, here ?

Reply:
I think original author didn't understand why we needed it. This original update is what you actually need to implement #151 correctly.